sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -199,12 +206,16 @@ def post_batch_committed(self, ledger_id, pp_time, committed_txns,
                                            self.token_state,
                                            len(committed_seq_nos_with_fees),
                                            state_root, txn_root_serializer.serialize(txn_root),
-                                           pp_time)
+                                           pp_time,
+                                           self.token_tracker)
             i = 0
             for txn in committed_txns:
                 if get_seq_no(txn) in committed_seq_nos_with_fees:
                     txn[FEES] = r[i]
                     i += 1
+            self.fee_txns_in_current_batch = 0
+        else:
+            self.token_tracker.un_committed.popleft()
```

Comment:
It would be better if we don't call internals of `token_tracker`, and call an API method.
I think `commit_batch` in `LedgerUncommittedTracker` doesn't require explicit arguments, and can calculate committed root hash and size from from the uncommitted queue.
This will allow just call `commit_batch()` here.